[11] Moonshot/Moonshot/Views/Mission/Crew List/MissionCrewListViewModel.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  MissionCrewListViewModel.swift
+//  Moonshot
+//
+//  Created by Joe Pham on 2022-08-09.
+//
+
+import Foundation
+
+/// Relies on its parent view model of type `MissionViewModel`,
+/// due to the way the original source JSON is structured:
+/// each Apollo mission's crew members is mapped to a
+/// corresponding astronauts name.
+///
+/// I couldn't pass a mission object to its initializer, because its crew
+/// is of `Mission`\'s nested type `Crew`, and not compatible
+/// with `CrewMember` type
```

Comment:
This is a bit hairy too, because it's different from the usage pattern of the other view models.
However, it's understandable this way too, because a crew list is dependent on its mission